### DISCUSSION TITLE
disable website `hexojs.github.io`

### DISCUSSION THREAD

**Discussion by uiolee** [hexojs/hexo#5409] [General]
![hexojs.github.io](https://github.com/hexojs/site/assets/22849383/d2050a74-8391-40ec-99fc-884c0e90eb3b)

------------
-----------

GitHub Pages [`hexojs.github.io`](https://hexojs.github.io/) is still accessible, and it retains outdated content. 

I suggest that we better turn off it to avoid misleading visitors.
Or redirect it to [`hexo.io`](https://hexo.io/)

**Comment by yoshinorin** [comment]
@uiolee 
I unarchived https://github.com/hexojs/hexojs.github.io repo and unpublished github pages. Now [hexojs.github.io](https://hexojs.github.io/) returns `404`.

Should we add an `index.html` to announce that the site has moved to `hexo.io`?

**Reply by renbaoshuo** [reply]
I think we should.

**Reply by uiolee** [reply]
feasible. pnpm does this too。 https://pnpm.github.io

**Reply by stevenjoezhang** [reply]
I have created an index.html in gh-pages branch, and it redirects to hexo.io
